git/repo/base.py

```diff
@@ -697,6 +697,19 @@ def _get_untracked_files(self, *args, **kwargs):
         finalize_process(proc)
         return untracked_files
 
+    def get_ignored(self, *paths):
+        """Checks if paths are ignored via .gitignore
+        Doing so using the "git check-ignore" method.
+
+        :param paths: List of paths to check whether they are ignored or not
+        :return: sublist of ignored paths
```

Comment:
What's the difference between `sublist` and `list`?

Reply:
What I meant is that I'm returning a `list` object which is a subset of the given paths which are ignored. Updated the documentation in order to make it clearer.